package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "vega-lite",
   "author": "Jeffrey Heer, Dominik Moritz, Kanit \"Ham\" Wongsuphasawat",
-  "version": "2.0.4",
+  "version": "2.1.0",
```

Comment:
We have to do that anyway because of some other breaking changes but I guess it's okay to do in the PR.

Reply:
Other breaking changes?

Reply:
#3275 makes some major changes to the schema.